web/sites/default/config/views.view.search_index.yml

```diff
@@ -54,7 +56,7 @@ display:
       pager:
         type: full
         options:
-          items_per_page: 20
+          items_per_page: 50
```

Comment:
Why the bump to 50 results? That will make the page load a bit slower.

Reply:
Reverted